source/site-administrators/studio/rte5-setup.rst

```diff
@@ -48,12 +48,21 @@ Let's take a look at an example of using one of the TinyMCE plugins.
 
 The default editor instance contains a menubar with most of the commonly used editing tools.  Sometimes, you want handy buttons available so you don't have to find the tool you need from the menubar.  We'll add a media button to our editor instance to be able to embed a YouTube video:
 
-1. Add the button name **media** to one of the toolbarItem tags: ``<toolbarItems1>``.  An **Insert/Edit Embedded Media** button |rteMediaBtn| will now be available for users of the RTE.
+1. Open the RTE (TinyMCE 5) configuration file in Studio by opening the **Sidebar**, then click on |siteConfig| -> *Configuration* -> *RTE (TinyMCE 5) Configuration*
+2. Add the button name **media** to one of the toolbarItem tags: ``<toolbarItems1>``.  An **Insert/Edit Embedded Media** button |rteMediaBtn| will now be available for users of the RTE.
```

Comment:
FYI
- On TinyMCE, buttons can be added through [`toolbar(n)`](https://www.tiny.cloud/docs/configure/editor-appearance/#toolbarn) or [`toolbar`](https://www.tiny.cloud/docs/configure/editor-appearance/#toolbar) but the rte config only supports toolbarItems(n). The new rte config — starting on v4 — will allow almost any tinymce config.
- Tiny's `toolbar(n)` are available only up to 4 through our `toolbarItems{1,2,3,4}`

Reply:
@alhambrav should we add some of Roy's notes to the docs?